omod/src/main/webapp/pages/ugandaEMRLogin.gsp

```diff
@@ -223,6 +223,7 @@
         background: #FFFFFB;
     }
     </style>
+    <script src="/${ui.contextPath()}/csrfguard" type="text/javascript"></script>
```

Comment:
Why are you adding this??

Reply:
@slubwama after pulling latest master, the login page could not load because of csrf error and on checking talk, this is the solution Kayiwa suggested. 
I added it and the login page was able to load.

Reply:
@semujju  there is a file that has to be added in the openmrs runtime directory.

Reply:
@slubwama which file is that?